docs/2.5. APIs - Query Parameters.md

```diff
@@ -485,71 +534,83 @@ NB: It is suggested that sort() and limit() are not implemented as paging and as
 ```
 
 ***Response***
 
 Headers
-```
+
+```http
 X-Ancestry-Generations: 4
 ```
 
 Payload
-* Returns Sources which list an ID of 'c1398579-15bc-468e-91ec-df5bbefe1cd3' in their 'parents' attribute array. These are the first generation ancestors.
-* For each first generation ancestor found, the ID of these Sources is used to look up the next generation in the 'parents' arrays of further Sources.
-* This process continues up to the generation limit returned in the X-Ancestry-Generations header. The server may refuse to honour a requested ''?query.ancestry_generations' value using a 400 error code if it is deemed to be too resource intensive.
+
+- Returns Sources which list an ID of 'c1398579-15bc-468e-91ec-df5bbefe1cd3' in their 'parents' attribute array. These are the first generation ancestors.
+- For each first generation ancestor found, the ID of these Sources is used to look up the next generation in the 'parents' arrays of further Sources.
+- This process continues up to the generation limit returned in the X-Ancestry-Generations header. The server MAY refuse to honour a requested ''?query.ancestry_generations' value using a 400 error code if it is deemed to be too resource intensive.
 
 ***Notes***
-* Responses should not include the Source with ID specified in the 'ancestry_id' parameter.
+
+- Responses SHOULD NOT include the Source with ID specified in the 'ancestry_id' parameter.
 
 **Example 2: Parents Of A Flow**
 
 ***Request***
-```
+
+```http
 GET /x-nmos/query/v1.1/flows?query.ancestry_id=ad14888a-3a98-444c-8aa8-4d87b77cbaa1&query.ancestry_type=parents&query.ancestry_generations=2
 ```
 
 ***Response***
 
 Headers
-```
+
+```http
 X-Ancestry-Generations: 2
 ```
 
 Payload
-* Returns Flows which are listed in the 'parents' array of the Flow identified by the ID 'ad14888a-3a98-444c-8aa8-4d87b77cbaa1'. These are the first generation parents.
-* For each first generation parent found, their 'parents' array is used to identify the next generation of parents to be returned.
-* This process continues up to the generation limit returned in the X-Ancestry-Generations header (in this case limited to 2 by a query parameter). The server may refuse to honour a requested ''?query.ancestry_generations' value using a 400 error code if it is deemed to be too resource intensive.
+
+- Returns Flows which are listed in the 'parents' array of the Flow identified by the ID 'ad14888a-3a98-444c-8aa8-4d87b77cbaa1'. These are the first generation parents.
+- For each first generation parent found, their 'parents' array is used to identify the next generation of parents to be returned.
+- This process continues up to the generation limit returned in the X-Ancestry-Generations header (in this case limited to 2 by a query parameter). The server MAY refuse to honour a requested ''?query.ancestry_generations' value using a 400 error code if it is deemed to be too resource intensive.
```

Comment:
Is that repeated single quote intended?

Reply:
Probably not,  and maybe we should move to back-ticks for this sort of thing anyway as we've done in more recent specs.